src/network.rs

```diff
@@ -42,9 +42,9 @@ const BONDING_MODES: [(u32,&str); 7] = [
     (BONDING_MODE_BALANCE_ALB,"balance-alb"),
 ];
 
-pub fn bonding_mode_to_string(mode: &u32) -> Result<String> {
```

Comment:
:eyes:

Reply:
I think this was https://rust-lang-nursery.github.io/rust-clippy/master/index.html#trivially_copy_pass_by_ref